src/dune

```diff
@@ -8,7 +8,7 @@
 
 (rule
   (target coq_elpi_builtins_arg_HOAS.ml)
-  (deps ../elpi/coq-arg-HOAS.elpi)
+  (deps ../elpi/coq-arg-HOAS.elpi (package elpi))
```

Comment:
Is this change an attempt from before the local copy of elpi2html?

Reply:
Unfortunately, no. This is used to circumvent a `dune` bug when dealing with plugins using OCaml dependencies in workspace builds. I don't think this hurts much, and I'm hoping @ejgallego and I can fix this soon-ish.